### ISSUE TITLE
AVT 2 - React Combo text fail color contrast

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3315]
## Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React

## Detailed Description
Use the Color Contrast Analyzer to test the background and foreground colors of the text against the background. 
The title and optional text fail on all combox examples, see screenshot below:
<img width="866" alt="Screen Shot 2019-07-09 at 1 18 35 PM" src="https://user-images.githubusercontent.com/21676914/60913470-192be780-a24d-11e9-8eb7-8453bba1864c.png">

In the items as components example, the red text in the list of options fails color contrast:
<img width="977" alt="Screen Shot 2019-07-09 at 1 38 47 PM" src="https://user-images.githubusercontent.com/21676914/60914471-36fa4c00-a24f-11e9-93ed-d3a71fb38956.png">

**Comment by dakahn** [comment]
@aagonzales 👋 Need some design input on this contrast violation.

**Comment by aagonzales** [comment]
> The title and optional text fail on all combox examples, see screenshot below

The title and optional text should be using the token `text-02`. In none of the themes is that value `#c3c3c3`. From my inspect of the react component in the white theme I'm seeing the value `#565656` which is correct and pass accessibility contrast.

http://react.carbondesignsystem.com/?path=/story/combobox--default
![image](https://user-images.githubusercontent.com/11670886/61900709-c5273100-aee3-11e9-89f7-de8f59705673.png)


@snidersd Have you over-rode the styles some where or are you maybe sampling the color wrong? I'm not sure where you're getting the `#c3c3c3` value.

**Comment by snidersd** [comment]
@aagonzales @dakahn  I just retested and it passes. FYI... I was using the Content Analyzer color picker, which appears to have an issue and it does not pick up the correct color code. If I type in the color passes. Thanks! Closing this issue.
